### DISCUSSION TITLE
Split up and tidy the readme/docs

### DISCUSSION THREAD

**Discussion by Callisto13** [actions/actions-runner-controller#1724] [Ideas]
👋 Hi!

I was wondering if you'd be open to a PR to organise the README a little better? Currently all on one very very long page, it is quite difficult to navigate, and you either have to be prepared to scroll a lot, or open the same page in tabs at different headings so you don't lose your place. Does anyone else have this problem?

I am more than happy to contribute splitting it up into easier-to-digest chunks inside dedicated `docs/getting-started/` and `docs/advanced-usage/` folders (or something like that) , just let me know. 😃

**Comment by mumoshu** [comment]
Hey! Thanks a lot for bringing this up. I do want to reorganize the docs. We have not yet decided the exact tool to be used or the structure to be followed, but would be great to discuss about it further in https://github.com/actions-runner-controller/actions-runner-controller/discussions/1722!

**Reply by Callisto13** [reply]
Ah nice! I searched for other discussions but that one did not come up 🤔
